### DISCUSSION TITLE
Privacy mode slows down the FPS (help wanted for new solution)

### DISCUSSION THREAD

**Discussion by sascha224** [rustdesk/rustdesk#3255] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug Description

When I enable the Privacy mode in the Client, the connection becomes very slow. The FPS decrease to 1-2 FPS, the remote PC becomes unusable. If I disable Privacy mode, all is getting back to normal.

### How to Reproduce

Enable the Privacy mode

### Expected Behavior

The Privacy mode should be as fast as the connection without it enabled.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.2.0

**Comment by rustdesk** [comment]
let follow parsec.

https://support.parsec.app/hc/en-us/articles/4421515077773-Privacy-Mode

![image](https://user-images.githubusercontent.com/71636191/231639711-98699766-c2c0-4b8b-b9a1-61b19b1b54a8.png)

by default, it is off per tv.

![image](https://user-images.githubusercontent.com/71636191/231641150-1dbe2f1c-a224-4a32-96ea-0bac5b22b2f4.png)

**Comment by rustdesk** [comment]
some clue on mac privacy mode

https://community.teamviewer.com/English/kb/articles/50966-teamviewer-black-screen

**Comment by rustdesk** [comment]
new privacy mode in 1.2.4 (nightly): https://github.com/rustdesk/rustdesk/pull/6406
